pages/linux/pkgfile.md

```diff
@@ -14,11 +14,11 @@
 
 - List all files provided by a package:
 
-`pkgfile --list {{package_name}}`
+`pkgfile --list {{package}}`
 
 - List only files provided by a package located within the `bin` or `sbin` directory:
```

Comment:
```suggestion
- List only files provided by a package located within the `/bin` or `/sbin` directory:
```

Reply:
But doesn't this refer to `bin` and `sbin` inside the package?

Reply:
Probably yes (https://man.archlinux.org/man/extra/pkgfile/pkgfile.1.en), will wait for confirmation from @acuteenvy.

Reply:
> But doesn't this refer to `bin` and `sbin` inside the package?

It does, but the package is unpacked to the same directory on the system so it doesn't really matter.

---

Also, both `/bin` and `/sbin` are symlinks to `/usr/bin` on Arch, and packages always use `/usr/bin`:

https://wiki.archlinux.org/title/Arch_package_guidelines#Directories
> - Packages should not contain any of the following directories:
> /bin
> /sbin
> ...

I'll make another suggestion.